p5-travel-app/src/server/server.js

```diff
@@ -13,3 +26,35 @@ app.use(bodyParser.json());
 
 // to use built files in dist folder
 app.use(express.static("./dist"));
+
+const PORT = 8080;
+app.listen(PORT, () => {
+  console.log(`CORS-enabled web server listening on port ${PORT}`);
+});
+
+const trips = [];
+
+// Routes
+
+app.get("/", (req, res) => {
+  res.status(200).send("./dist/index.html");
+});
+
+app.post("/forecast", async (req, res, next) => {
+  if (req.body.endpoint !== " ") {
+    const endpoint = req.body.endpoint;
+    try {
+      const response = await fetch(endpoint);
+      if (response.ok) {
+        const jsonRes = await response.json();
+        res.status(201).send(jsonRes);
+      }
+    } catch (error) {
+      console.log(error);
```

Comment:
console warn

Reply:
done in [77541ac](https://github.com/leemoispace/Udacity-Front-End-WebDeveloper/pull/9/commits/77541acca892e61c7f81d34c2e31e59fbb1a7011)